third_party/googleapis/google/api/annotations.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015, Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api;
+
+import "google/api/http.proto";
```

Comment:
Would you consider changing this import?

```diff
-import "google/api/http.proto";
+import "github.com/gengo/grpc-gateway/third_party/googleapis/google/api/http.proto";
```

The motivation is that, for generating .pb.gw.go files in user programs, users would typically do something like this, as described in README:

```
protoc -I/usr/local/include -I. -I$GOPATH/src \
 --grpc-gateway_out=logtostderr=true:. \
 path/to/your_service.proto
```

That would cause the `google/api/http.proto` import to not be found. It would require users to add an additional include path directive:

```
protoc -I/usr/local/include -I. -I$GOPATH/src \
 -I$GOPATH/src/github.com/gengo/grpc-gateway/third_party/googleapis \
 --grpc-gateway_out=logtostderr=true:. \
 path/to/your_service.proto
```

Which is extra work that can be eliminated. What do you think?

To make it work, all imports `google/api/http.proto` would be need to be updated to `github.com/gengo/grpc-gateway/third_party/googleapis/google/api/http.proto`. It also seems to be more in spirit of Go to use a full import path rather than a relative one.